src/Raven.Studio/typescript/viewmodels/database/settings/clientConfiguration.ts

```diff
@@ -33,6 +35,10 @@ class clientConfiguration extends viewModelBase {
 
         this.bindToCurrentInstance("saveConfiguration", "setReadMode");
 
+        this.canNavigateToServerSettings = ko.pureComputed(() => {
```

Comment:
no need for wrapping in computed I guess? this.canNavigateToServerSettings = accessManager.default.isClusterAdminOrClusterNodeClearance ?

Reply:
done